build.gradle.kts

```diff
@@ -3,6 +3,21 @@ plugins {
     base
     // Kotlin plugin should be loaded only once
     id("com.gevamu.kotlin-common-conventions") apply false
+    // Generate api reference with dokka
+    id("org.jetbrains.dokka") version "1.7.20"
```

Comment:
I think we need to use the same version as kotlin version

Reply:
You can set manual version to the `0.7.5`. See https://mvnrepository.com/artifact/org.jetbrains.kotlinx/kotlinx-html-jvm/0.7.5